src/js_stream.cc

```diff
@@ -87,24 +87,40 @@ bool JSStream::IsAlive() {
 bool JSStream::IsClosing() {
   HandleScope scope(env()->isolate());
   Context::Scope context_scope(env()->context());
-  return MakeCallback(env()->isclosing_string(), 0, nullptr)
-      .ToLocalChecked()->IsTrue();
+  TryCatch try_catch(env()->isolate());
+  Local<Value> value;
+  if (!MakeCallback(env()->isclosing_string(), 0, nullptr).ToLocal(&value)) {
+    FatalException(env()->isolate(), try_catch);
```

Comment:
`FatalException()` can return so this should probably be followed by an `ABORT()` or something.  The unguarded `value->IsTrue()` below is not safe, at any rate.

Reply:
@bnoordhuis Yea, thanks. 👍 I’ve added an extra `return true;` here and `UV_EPROTO` as the default return value for the other places, that’s what `tls_wrap.cc` uses as a generic failure code (but feel free to let me know of a better choice) – it shouldn’t make any difference in practice, I guess.